src/metabase/models/serialization/util.clj

```diff
@@ -408,7 +408,7 @@
   "Given the :parameter field of a `Card` or `Dashboard`, as a vector of maps, converts
   it to a portable form with the CardIds/FieldIds replaced with `[db schema table field]` references."
   [parameters]
-  (map ids->fully-qualified-names parameters))
```

Comment:
this is needed because for some reason `ingest-in-memory` during test doesn't realize this lazy seq and we got a `nil` serdes path.

Reply:
nit: use mapv?

Reply:
And I would add this as a comment. It's not clear why `export-parameters` needs to be a vector, but `export-parameter-mappings` doesn't.